src/TwigDebugBundle/Twig/Node/Twig_Node_Include_Decorator.php

```diff
@@ -51,7 +51,7 @@ public function getIterator(): \Traversable
         return $this->original->getIterator();
     }
 
-    public function getNodeTag(): string
+    public function getNodeTag(): ?string
```

Comment:
In this class I changed the actual PHP type information to account for nullable returns that may happen. I did this because type hints existed before.

Note: This leads to actual behavioural changes: Before this, the methods would do an implicit string cast and return `''` in case of `null`. Now, it returns `null`. Adding an explicit type-cast would more accurately reflect the previous behaviour.

Reply:
That's a good question: It would have been ok with the minimum PHP version of 7.4 declared for the 7.1.x branch. With moving this to the 7.0.x branch it is not ok anymore. I'll revert the changes and add explicit type-casts. That way the type annotations are valid while the behaviour does not change: We are just moving the cast from being implicit to being explicit

Reply:
Yes, It would have been: chameleon 7.1.x requires a minimum of PHP7.4, which supports nullable types. However, with moving this PR to the 7.0.x branch, the entire PHP-Type hint is gone and is now a docblock